### PR TITLE
ECOSYS-14 Parsing Java version with text suffix causes NumberFormatException

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

File: plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java [hunk 1]
```diff
@@ -25,23 +25,25 @@
...
+ *
  * <p>
  * Example: <br>
- * 
+ *
  * 1.6.0_u14 == major = 1 minor = 6, subminor = 0, update = 14
  *
  * @author bnevins
  */
 public final class JDK {
```
Comment (MarkWareham) [inline plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java]: There's a lot of intricate logic in this class. I would suggest some unit tests

Comment (jGauravGupta) [reply plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java]: Created the unit test for JDK version parsing but eclipse plugin build is not picking it.

Comment (jGauravGupta) [reply plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java]: @dmatej @MarkWareham help needed on this.

Comment (dmatej) [reply plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java]: Run As -> JUnit Plug-in Test?

Comment (dmatej) [reply plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java]: Eh, but the class is not a plugin ... hmm. I'm not sure how to run unit tests inside eclipse extension project ...

Comment (jGauravGupta) [reply plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java]: Created ECOSYS-88 to address the ant build issue.

**Pull request merged**